### DISCUSSION TITLE
Melhor tecnologia para um ecommerce

### DISCUSSION THREAD

**Discussion by pedroribeirodev** [frontendbr/forum#1628] [Perguntas e Respostas]
Estou criando uma api para um ecommerce, a API tá sendo feita com AdonisJS.

Vi muitos artigos sobre o NextJS o GatsbyJS.

Queria saber de vocês qual a melhor opção para fazer o front end do meu ecommerce levando em conta que estou criando uma API REST com o AdonisJS

Obrigado!

**Comment by SrVieira** [comment]
Você tem que analisar algumas questões e pontua-las. O nextjs é SSR enquanto o gatsby é SSG. Se o seu site será somente uma vasta coleção de páginas estáticas e não teria uma atualização massiva eu iria de Gatsby caso queira uma melhor perfomance e manutenção eu iria de Next. Segue um link interessante: https://dev.to/jameesy/gatsby-vs-next-js-what-why-and-when-4al5

**Comment by eliseumds** [comment]
Gostaria de adicionar que a galera do Next.js está discutindo implementar SSG também: https://github.com/zeit/next.js/issues/9524. É opt-in e o desenvolver vai poder implementar uma solução híbrida, aparentemente. Bem bacana.

**Comment by Juuhz** [comment]
Eu fiz o ecommerce da minha startup em Next.Js

https://www.cardesivo.com.br/

Resultado ficou muito bom, recomendo.
